cfy_manager/components/components_dependencies.py

```diff
@@ -25,7 +25,12 @@
     'python-setuptools': 'required by python',
     'python-backports': 'required by python',
     'python-backports-ssl_match_hostname': 'required by python',
-    'openssh-server': 'required by the sanity check'
+    'openssh-server': 'required by the sanity check',
+    'prometheus': 'required for monitoring (status reporter agent)',
+    'blackbox_exporter': 'used for monitoring HTTP services',
+    'node_exporter': 'used for monitoring VMs',
+    'postgres_exporter': 'used for monitoring PostgreSQL',
+    'rabbitmq_exporter': 'used for monitoring RabbitMQ',
```

Comment:
this isn't required, is it? this "dependencies" list is for declaring pre-requisites that need to be installed before running the installer;
things that our installer installs shouldn't be here, but should be in sources.py instead (split into db/queue/manager similarly)

Reply:
it _is_ required.  without those installation script raises an exception complaining there are not keys `prometheus` (and others) available and installation fails.

Reply:
OK, last solution (RPMs instead of curl) made my previous comment obsolete.